rj_config/src/config_client/config_client_node.cpp

```diff
@@ -0,0 +1,6 @@
+#include <config_client/config_client_node.h>
+
+namespace config_client {
+ConfigClientNode::ConfigClientNode(const std::string& name)
```

Comment:
What is the purpose of this node? I'm confused as to why it's necessary - is it just a test?

Reply:
It's so that something like `Processor` which isn't itself a `rclcpp::Node` can use this, since `config_client::ConfigClient` basically tacks on subscribers to an existing node.

Reply:
ACK, so it's a temporary measure. Can you document it as such?

Reply:
Sure.